hack/lib/version.sh

```diff
@@ -161,6 +161,7 @@ kube::version::ldflags() {
       "-X '${KUBE_GO_PACKAGE}/vendor/k8s.io/client-go/pkg/version.${key}=${val}'"
       "-X '${KUBE_GO_PACKAGE}/vendor/k8s.io/kubectl/pkg/version.${key}=${val}'"
       "-X '${KUBE_GO_PACKAGE}/cmd/kubeadm/app/version.${key}=${val}'"
+      "-X '${KUBE_GO_PACKAGE}/vendor/k8s.io/component-base/version.${key}=${val}'"
```

Comment:
@stealthybox this needs follow-ups in all of our components.

Reply:
i.e. use this, not pkg/version

Reply:
https://github.com/kubernetes/kubernetes/issues/81731

Reply:
Yeah, we have like a bazillion copies of this thing currently everywhere.